<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::East</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="dialTab">
       <attribute name="title">
        <string>Dialpad</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="0" rowspan="2">
         <widget class="QWidget" name="numDisp" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QComboBox" name="cbNumbers"/>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="dispNum"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="keypad" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <item row="4" column="3">
            <widget class="QPushButton" name="k4">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="4" column="6">
            <widget class="QPushButton" name="k5">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="4" column="9">
            <widget class="QPushButton" name="k6">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item row="7" column="6">
            <widget class="QPushButton" name="k8">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item row="7" column="9">
            <widget class="QPushButton" name="k9">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
           <item row="9" column="3">
            <widget class="QPushButton" name="kstar">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>*</string>
             </property>
            </widget>
           </item>
           <item row="9" column="6">
            <widget class="QPushButton" name="k0">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="9" column="9">
            <widget class="QPushButton" name="khash">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>#</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="k1">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="3" column="6">
            <widget class="QPushButton" name="k2">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="3" column="9">
            <widget class="QPushButton" name="k3">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QPushButton" name="k7">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btnText">
             <property name="text">
              <string>Text</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnCall">
             <property name="text">
              <string>Call</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="contactsTab">
       <attribute name="title">
        <string>Contacts</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTreeView" name="contactsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="inboxTab">
       <attribute name="title">
        <string>Inbox</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTreeView" name="inboxView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="loginDetails">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>96</width>
             <height>101</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Login details</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLineEdit" name="textUsername">
              <property name="placeholderText">
               <string>Username: example@gmail.com</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="textPassword">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="loginButton">
              <property name="text">
               <string>Login</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="proxyDetails">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>420</width>
             <height>240</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Proxy details</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="1">
             <widget class="QCheckBox" name="useSystemProxy">
              <property name="text">
               <string>Use System Proxy</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="enableProxy">
              <property name="text">
               <string>Enable Proxy</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QFrame" name="frameProxyServerPort">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Server</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="proxyServer">
                 <property name="placeholderText">
                  <string>Proxy server name or address</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Port</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="proxyPort">
                 <property name="placeholderText">
                  <string>Proxy port</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>User</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="proxyUser">
                 <property name="placeholderText">
                  <string>Username for proxy</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="proxyPass">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="placeholderText">
                  <string>Password for proxy</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QCheckBox" name="proxyAuthRequired">
                 <property name="text">
                  <string>Requires authentication</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QDialogButtonBox" name="proxyButtonBox">
              <property name="standardButtons">
               <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="aboutDetails">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>112</width>
             <height>112</height>
            </rect>
           </property>
           <attribute name="label">
            <string>About</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>497</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>textPassword</sender>
   <signal>editingFinished()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
